spec/models/like_spec.rb

```diff
@@ -1,5 +1,19 @@
 require 'rails_helper'
 
 RSpec.describe Like, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it 'tests that Like model is created correctly' do
+    like = Like.new
+    like.build_user(name: 'Orçun')
+    like.build_post(text: '')
+    expect(like).to be_valid
+  end
+
+  it 'updates a posts likes correctly' do
+    user = User.new(name: 'Tom', posts_counter: 0)
+    user2 = User.create!(name: 'Lilly', posts_counter: 0)
+    post = user2.posts.create!(title: 'Post1', text: 'This is a post', likes_counter: 0, comments_counter: 0)
+    post.likes.create!(user:)
```

Comment:
- [ ] It gives me syntax errors when I'm running `rspec`. I think this line should be `post.likes.create!(user: user)`. Please fix that.

![Screenshot from 2022-03-05 01-39-36](https://user-images.githubusercontent.com/22434660/156834365-214b9a21-02a7-4623-953d-b8a3127ffeed.png)